### PR TITLE
NMS-12617: XSS security issues

### PR DESCRIPTION
**Pull request by patrick-schweizer**
This PR addresses xss issues in jsp includes.
There are still open questions, see ticket: https://issues.opennms.org/browse/NMS-12617

### CODE REVIEW

File: core/web-assets/src/main/assets/static/load-assets.jsp [hunk 1]
```diff
@@ -60,7 +65,7 @@ if (locator == null) {
                         .append(resource.getPath())
                         .append("?v=").append(lastModified)
                         .append("\"");
-                    if (mediaString != null && mediaString.trim() != "") {
+                    if (mediaString != null && mediaString.trim().equals("")) {
```
Comment (RangerRick) [inline core/web-assets/src/main/assets/static/load-assets.jsp]: the second half of this should be negated: `if (mediaString != null && !mediaString.trim().equals("")) {`

Comment (patrick-schweizer) [reply core/web-assets/src/main/assets/static/load-assets.jsp]: :man_facepalming: thanks @RangerRick

**Comment by RangerRick** [comment]
Shouldn't this be targeting `foundation-2019` as well, so it makes it into Meridian?

**Comment by patrick-schweizer** [comment]
Replaced by: https://github.com/OpenNMS/opennms/pull/2953

**Pull request closed without merging**